### ISSUE TITLE
France - programme-television.org

### ISSUE THREAD

**Issue by Lynne12** [Robinmcgvr/Kapow-Data#269]
@Robinmcgvr Ciné+ Premier ceases on Kablam on 15 October.

https://en.wikipedia.org/wiki/Cin%C3%A9%2B_OCS

**Comment by Robinmcgvr** [comment]
Thanks Lynne, I 've asked Matteo to add OCS or cine+OCS to our main sites

**Comment by Robinmcgvr** [comment]
Thanks I've asked Matteo to add it to the relevant sites

**Comment by Lynne12** [comment]
@Robinmcgvr thanks, can you please let me know when this is done, so I can set up the channel for loading and also confirm if there will be backdata, as I have data missing from the current loading week of online scheduling, 15-21 October 2024.

**Comment by Robinmcgvr** [comment]
data is still there under original channel name on programm-tv.net up to 23/10

**Comment by Lynne12** [comment]
Data loaded to fill gap.  Will wait to hear when the new loading site is ready.

**Comment by Lynne12** [comment]
@Robinmcgvr any news on this, please.  We have no data loaded for CINC from 22nd October.

**Comment by Robinmcgvr** [comment]
Ok I shall look into it, this has slipped through both our nets

**Comment by Robinmcgvr** [comment]
Could I ask that when you raise an issue affecting a particular channel or channels that you use the actual channel names please?
Makes things easier.

**Comment by Robinmcgvr** [comment]
This channel fka (Cine Cine premier) is now known as OCS on that and other sources and has data from 28/10 onwards

**Comment by Lynne12** [comment]
> Could I ask that when you raise an issue affecting a particular channel or channels that you use the actual channel names please? Makes things easier.

I used the abbreviated channel name CINC, from the Studio and Atelier dropdown

**Comment by Lynne12** [comment]
Just to confirm there will still be a gap in the data from 22-27 October.

**Comment by Lynne12** [comment]
Gap in data loaded from programme-tv.net

**Comment by Lynne12** [comment]
@Robinmcgvr the following dates are missing, 8-13 January 2025, for all channels.

**Comment by Robinmcgvr** [comment]
i am trying to see if Matteo is able to work with the format of the tv guide on this site as it has changed a bit
We do have the data available from a back up source, there is a problem with getting this one added to Kablam as it duplicates another sources name (one we use for belgium) but if we need to shift to this source I will get that issue fixed

**Comment by Lynne12** [comment]
@Robinmcgvr the missing dates now include 14-20 January 2025.

**Comment by Robinmcgvr** [comment]
Actually the only other site that gives us all the channels we need is tv-programme.telecablesat.fr - there are a couple of dates missing from here too, but this might be the best we can do. So maybe load from there?

**Comment by Lynne12** [comment]
@Robinmcgvr we had a lot of trouble, in the past, with tv-programme.telecablesat.fr, missing dates etc, so would rather wait until you explore the current loading website fully.   As you mentioned above there are still dates missing for the date range in question. In addition, on a quick check tv-programme.telecablesat.fr does not have CINE CINEMA PREMIER (CINC) data, unless there is a name change I am unaware of.

**Comment by Robinmcgvr** [comment]
@Lynne12 
See revised message
Cinc is OCS or cine+OCS now on some sources
I will see what we can do about the tv-programme.com downloads to make them loadable in Kablam , as you have probably seen we had a major kapow issue yesterday , so once that is sorted out we might finally be able to fix the france problems.
I guess we have some more time to see what can be done as we won;t be entering Jan data for a while yet

**Comment by Lynne12** [comment]
@Robinmcgvr yes will wait, for now.  Of the date range missing from programme-television.org, tv-programme.telecablesat.fr also has 9 days missing, so there would still be a lot of gaps.
Just to say that we are going to making advances on the Online Scheduling processing, as we are very short of verified data.  November is nearly complete and then we will be rapidly making our way through December (unless the delayed verified data suddenly appears).

**Comment by Robinmcgvr** [comment]
@Lynne12 
We have the newer site in Kablam now  it is tv-programme.com - covers all the channels and dates we need.
As it has only just moved to kablam there are no dates showing against the data in the channel selection drop down but if you add the dates into the date selector the data will appear.

**Comment by Lynne12** [comment]
@Robinmcgvr are we making a permanent switch?

**Comment by Robinmcgvr** [comment]
Let's see what the data looks like loaded in compared to the previous site which is now fixed (but not recovered) I can see that there is a little bit of 10/01 missing, looks like the only bit , so we're going to have to plug the gaps in morning of 10/01 manually . It looks like that is the only day like that.  So shall we load from here to plug the gap and then review the data?
You could load the whole of Jan from that source if it makes it easier ....

**Comment by Lynne12** [comment]
I will reload from 1st January, as there appears to be gaps in some of the days loaded already.

**Comment by Lynne12** [comment]
Just to keep a full note 21-24 January is also missing from the programme-television.org website.  I am currently loading data for all channels from tv-programme.com, 1-27 January.

**Comment by Lynne12** [comment]
@Robinmcgvr data is loaded, if you could let me know if you are happy with it.

**Comment by Lynne12** [comment]
@Robinmcgvr can you please confirm which loading source we will use, going forwards for CINC-Cine Cinema Premier, please.

**Comment by Lynne12** [comment]
@Robinmcgvr I am waiting on loading this data, as we will soon be moving onto January scheduling.  If you are able to confirm the website, please.

**Comment by Robinmcgvr** [comment]
@Lynne12 
Ok I am looking at this now and there is something a bit weird going on unfortunately
I will see if I can get to the bottom of it - but I would say hold off on scheduling france Cinc for now (sorry)

**Comment by Robinmcgvr** [comment]
We will have to unload that data for January from source tv-programme.com as it stands now it is not fit for purpose.
I don't know why we keep having to have a saga over this french channel/data - at least it's only one channel affected now (for scheduling anyway) 
Currently the best data source for this channel looks like programm-tv.net - I know this one has had its issues too, but so far it's looking like the best data we currently have
I have sent Matteo screengrabs and files to demonstrate the issue, and hopefully he can do something about it.
If not it will have to be the programme-tv.net data we use for the time being

**Comment by Lynne12** [comment]
@Robinmcgvr I will get the data unloaded, for January.  Shall I wait on you hearing from Matteo before setting up either of the alternatives?

**Comment by Robinmcgvr** [comment]
That depends how long you can wait, it will take at least a week for Matteo to investigate and fix (if not more), so if you want to act more quickly then the programme-tv.net might be the better bet as it looks alright.

**Comment by Lynne12** [comment]
Okay Robin, I will take a view when we are nearing January scheduling, for France.  Many thanks.

**Comment by Robinmcgvr** [comment]
re. programme-tv.net if we do switch to this site for loading, there is often a lag of up to a week on the data , so keep that in mind when loading

**Comment by Robinmcgvr** [comment]
Matteo has fixed the data from tv-programme.com but not retrospectively, only going forwards from 11/02

**Comment by Robinmcgvr** [comment]
I think I would recommend loading the January and February - from programme-tv.net - it looks like the best consistent data at present and may as well stick with this one until the next problem ...

**Comment by Lynne12** [comment]
Are we going to stick with tv-programme.com, going forward, so that I can set up the data loader?

**Comment by Lynne12** [comment]
1-31 January 2025 and 1-17 February loaded from programme-tv.net.  Remainder of Feb will be loaded from the same website.
From 1 March 2025 tv-programme.com will be used for loading.
